test/org/rascalmpl/util/maven/MavenResolverTest.java

```diff
@@ -97,6 +97,10 @@ public void rascalPomHasRightDependencies() throws ModelResolutionError {
 
         var maybeCapsule = locate(resolved, "capsule");
         assertTrue("Vallang should depend on capsule", maybeCapsule.isPresent());
+        Path sha1Path =
+
+        tempRepo.resolve(Path.of("io", "usethesource", "vallang", "1.0.0-RC15", "vallang-1.0.0-RC15.jar.sha1"));
```

Comment:
nit: remove the newlines between decl and init?

maybe we should also have a similar check to make sure the jar got downloaded?